api/specs/common/schemas/node-meta-v0.0.1.json

```diff
@@ -248,6 +248,11 @@
                 true
               ]
             },
+            "unit": {
+              "title": "Unit",
+              "description": "Units of this input value, if a physical quantity",
+              "type": "string"
```

Comment:
what about nullable here? what happens with the old services that do not follow the new schema? does that work?

Reply:
I do not want it to be nullable but optional. 
If it is not set, it should NOT appear.
Now, in the model instance (on the server side), when the model is received, it will create an object whose ``unit`` attribute will be ``None``.  BUT, when returned in a response, it should NOT be set (i will make sure this is the case)

Reply:
👍